core/src/main/java/com/datastrato/gravitino/catalog/CapabilityHelpers.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog;
+
+import static com.datastrato.gravitino.rel.Column.DEFAULT_VALUE_NOT_SET;
+
+import com.datastrato.gravitino.connector.capability.Capability;
+import com.datastrato.gravitino.rel.Column;
+import com.datastrato.gravitino.rel.TableChange;
+import com.google.common.base.Preconditions;
+import java.util.Arrays;
+
+public class CapabilityHelpers {
+
+  public static Column[] applyCapabilities(Column[] columns, Capability capabilities) {
```

Comment:
It is just one capability, why do you need to use plural?

Reply:
This method will apply multiple capabilities to the column, such as default value, not null, case sensitivity, and so on. Therefore I use plural